src/main/java/io/github/jancieslak/tba/repository/ScreeningRepository.java

```diff
@@ -0,0 +1,21 @@
+package io.github.jancieslak.tba.repository;
+
+import io.github.jancieslak.tba.model.Screening;
+import io.github.jancieslak.tba.model.ScreeningRoom;
+import io.github.jancieslak.tba.model.Seat;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface ScreeningRepository extends JpaRepository<Screening, Long> {
+    @Query(nativeQuery = true, value = "select * from Screening s inner join Movie m on s.movie_id = m.id where s.from_date_time between :beginFrom and :beginTo")
```

Comment:
dlaczego decyzja o native query?

Reply:
Tylko w native query udało mi się porównać LocalDateTime. Szukałem jakiejś odpowiedzi ale niestety nie znalazłem. Jedyne co znalazłem i mogło być odpowiedzią to converter z typu LocalDateTime na inny który byłby obsługiwany.